client/src/components/Cell/CellMain.tsx

```diff
@@ -84,21 +108,26 @@ const CellMain = ({
           {t('rankUp')}
         </FantasyButton>
       )}
+
       {owner && (brute.xp < xpNeededForNextLevel ? fightsLeft > 0 ? (
         <Stack spacing={1} sx={{ alignItems: 'center', mt: 1 }}>
           <Text bold sx={{ pl: 1 }}>{t('callToFight')}</Text>
-          <Link to={`/${brute.name}/arena`}>
-            <StyledButton
-              sx={{
-                height: 72,
-                width: 218,
-              }}
-              image={`/images/${language}/cell/arena.gif`}
-              imageHover={`/images/${language}/cell/arena-hover.gif`}
-              shadow={false}
-              contrast={false}
-            />
-          </Link>
+          <Box sx={{ display: 'flex' }}>
+            { smallScreen && <NavigateBeforeIcon onClick={() => swipeBrute('left')} sx={{ ...arrowProps, left: 0, opacity: smallScreen && bruteList && bruteList[bruteList.indexOf(brute.name) - 1] ? 1 : 0 }} />}
```

Comment:
If you want to display something conditionally, don't use opacity, use conditions instead:
```tsx
{yourCondition && (
  <YourConditionalComponent />
)}
```